src/mcover/coverage/client/LcovPrintClient.hx

```diff
@@ -258,13 +323,19 @@ class LcovPrintClient implements CoverageReportClient {
 	}
 
 	function appendCoverageFile(text:String) {
-		var file:FileOutput = sys.io.File.append(lcovFileName);
-		file.writeString(text.toString());
-		file.close();
+		#if nodejs
+			var content:String = sys.io.File.getContent(lcovFileName);
```

Comment:
Could you avoid re-reading the file for every output? or maybe use directly the `fs.appendFile` API.

Reply:
makes sense, I'll look into it